.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Feature request
+about: Suggest an idea or a feature for this project
+title: ''
+labels: feature request
+assignees: ''
```

Comment:
given that issues don't typically get a notification for codeowners, do we want to test if this field takes a team name and notifies/assigns them?

Reply:
Wouldn't that mean everyone on the team would be notified of any feature request/bug in any repo? That seems like too much noise.